internal/controller/controller.go

```diff
@@ -98,20 +106,15 @@ func StartControllers(ctx context.Context, config *rest.Config, logger logr.Logg
 
 	sink := newConfigSink(clientForConfigSink, kubeForConfigSink, logger, sinkChan)
 
-	if leaderElection {
-		logger.Info("Waiting to become the leader")
-		<-mgr.Elected()
-		logger.Info("Became the leader")
-	}
```

Comment:
i believe this is needed?

Reply:
I am pretty sure this is handled inside of the controller manager.

Reply:
ok that's good to know... appreciate your expertise here